tests/conftest.py

```
@@ -0,0 +1,92 @@
+@pytest.fixture
+def sample_hook_input():
+    """Sample input data for hook testing."""
+    return {
+        "hook_event_name": "PreToolUse",
+        "tool_name": "Bash",
+        "tool_input": {
+            "command": "git status",
+            "description": "Check git status",
+        },
+        "session_id": "test-session-12345678",
+        "cwd": "/tmp/test-project",
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/test-project"

(S108)